### DISCUSSION TITLE
Does not work on Windows 7

### DISCUSSION THREAD

**Discussion by flawrist** [Jacalz/rymdport#130] [Ideas]
The launch of application is interrupted without an error message from which the user could get an idea of what went wrong and guess how to fix it. Consider adding one.

**Comment by Jacalz** [comment]
I already do show any errors in the gui when the application has started. However, if the application crashes or the graphical window can't be created (happens quite often on Windows sadly), I obviously can't show anything to the user. Windows is a strange operating system and won't print stdout to a terminal unless you build the application to open a terminal on startup (which is very annoying for users). Implementing https://github.com/Jacalz/rymdport/issues/88 should hopefully make things easier.

Can you open a bug report instead so we can try to get your problem fixed?
